### PR TITLE
Reformat in data_type function closest_valid_dtype 

### PR DESCRIPTION
**Pull request by Amrabdelhamed611**
Reformat  in ivy functional data_type.py for closest_valid_dtype  function issue  #2321.

### CODE REVIEW

**Comment by Amrabdelhamed611** [comment]
#2321

File: ivy/functional/ivy/data_type.py [hunk 1]
```diff
@@ -708,17 +708,49 @@ def _check_float64(input) -> bool:
...
+    >>> yType = ivy.closest_valid_dtype(xType)
+    >>> print(yType)
+    int8
+
+    With :code:`ivy.Native_dtype` inputs:
+
+    >>> xType = ivy.native_uint16
+    >>> yType = ivy.closest_valid_dtype(xType)
+    >>> print(yType)
+    uint16
```
[the remark this answers is not in the document]
Comment (Amrabdelhamed611) [reply ivy/functional/ivy/data_type.py:745]: when i change uint16  to  <dtype:'uint16'> my docs test fails 

Expected:
    <dtype:'uint16'>
Got:
    uint16

[the remark this answers is not in the document]
Comment (Amrabdelhamed611) [reply ivy/functional/ivy/data_type.py]: i added the sopces on commit and <dtype:'uint16'> on different  commit to roll it back if you want

**Review by VedPatwardhan — APPROVED** [review]
lgtm! Thanks for contributing 🙂

**Pull request merged**